From 799a7dd38ec70a3913cc405feffbbad7c458df97 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Wed, 11 Jul 2012 16:28:30 +0800
Subject: [PATCH 027/310] ENGR00215859-1 BOARD_HAVE_VPU set to false on imx6sl

BOARD_HAVE_VPU set to false on imx6sl as no VPU in this chip

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 imx6/soc/imx6sl.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/soc/imx6sl.mk b/imx6/soc/imx6sl.mk
index 0898398..b40184e 100644
--- a/imx6/soc/imx6sl.mk
+++ b/imx6/soc/imx6sl.mk
@@ -3,7 +3,7 @@
 #
 
 BOARD_SOC_TYPE := IMX6SL
-BOARD_HAVE_VPU := true
+BOARD_HAVE_VPU := false
 HAVE_FSL_IMX_GPU2D := true
 HAVE_FSL_IMX_GPU3D := false
 HAVE_FSL_IMX_IPU := false
-- 
1.8.0

